src/index.html

```diff
@@ -11,8 +11,58 @@
       rel="stylesheet"
       href="./styles/index.scss"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <section class="catalog">
+      <div
+        class="catalog-card"
+        data-qa="card"
+      >
+        <img
+          class="catalog-card_image"
+          src="./images/imac.jpeg"
+          alt="computer monitor"
```

Comment:
```suggestion
          alt="APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)"
```
`alt` attribute should describe image as well as possible